doc/debugger.rst

```diff
@@ -1875,7 +1876,7 @@ GetBytecode request (0x21)
 
 Format::
 
-    REQ <int: 0x21> [<int: level> OR <obj: target> OR <heapptr: target>] EOM
+    REQ <int: 0x21> [<int: level | obj: target | heapptr: target>] EOM
```

Comment:
The ones that are inconsistent is because the contract is "(this 1 value) OR (these 2 values)".  So it got a different notation than "(1 value with either this or that type)"